### ISSUE TITLE
Camera Thumbnail will not show up if no images on device

### ISSUE THREAD

**Issue by pathaderavi** [zhangao0086/DKImagePickerController#576]
Camera thumbnail does not show up in the gallery if there are no images on the current device (Only for Photos mode).

**Comment by zhangao0086** [comment]
Hi @pathaderavi It's a little odd, there are no `Camera Roll` and `Favorites` albums on your device?

![simulator screen shot - ipad air 2 - 2019-02-26 at 21 25 47](https://user-images.githubusercontent.com/7742344/53415876-41891700-3a0d-11e9-87d4-74139f73c99d.png)

**Comment by Rahulrvarma22** [comment]
I am facing the same issue. Camera image is not loaded. Shows up on first time but later no image in the cell.

```
    public class func imageForResource(_ name: String, stretchable: Bool = false, cacheable: Bool = false) -> UIImage {

...

        let bundle = Bundle.imagePickerControllerBundle()
...
}
```
**Printing description of bundle:**
NSBundle </Users/app-path/appname.app/**Frameworks/DKImagePickerController.framework/DKImagePickerController.bundle> (not yet loaded)**

Please see the attached screenshot.

![simulator screen shot - iphone xs max - 2019-03-03 at 19 01 47](https://user-images.githubusercontent.com/6213405/53695857-d82b4e80-3de6-11e9-864e-0a2ad8920046.png)

**Comment by pathaderavi** [comment]
> Hi @pathaderavi It's a little odd, there are no `Camera Roll` and `Favorites` albums on your device?
> 
> _From iPad Air Simulator:_
> ![simulator screen shot - ipad air 2 - 2019-02-26 at 21 25 47](https://user-images.githubusercontent.com/7742344/53415876-41891700-3a0d-11e9-87d4-74139f73c99d.png)

The issue is not with Camera Roll or Favorite tile, but the "Camera" tile which opens the camera.

**Comment by zhangao0086** [comment]
@pathaderavi I know the issue is not with Camera Roll or Favorite. This issue means that there are no any albums on the device, and the `Camera` tile needs to be displayed in an album.

Maybe the logic of the `Camera` tile display should not be associated with the albums: https://github.com/zhangao0086/DKImagePickerController/commit/ec0e70e8908955edbb2be55e8f3a68b1c5863f8b

**Comment by Rahulrvarma22** [comment]
Hi @zhangao0086  will this solve my problem (https://github.com/zhangao0086/DKImagePickerController/issues/576#issuecomment-469023359) also of the camera image not showing up on the 1st tile?

**Comment by zhangao0086** [comment]
@Rahulrvarma22 No. It's a different issue. Please try to clean your project with Shift-Alt-Cmd-K and delete the derived data.
